Kitodo-DataAccess/src/test/java/org/kitodo/dataaccess/format/xml/XMLWriterTest.java

```diff
@@ -0,0 +1,112 @@
+package org.kitodo.dataaccess.format.xml;
```

Comment:
Missing project license header - please add the license header!

Reply:
Was added—It seems that GitHub does not automatically update, I don’t know why.
→ We have to check checkstyle setup as it should refuse missing license headers.

Reply:
Checkstyle checks for missing license header in java classes, but... we don't run checkstyle on test resources.